SRC/NETWORK/hget.c

```diff
@@ -422,13 +444,25 @@ void InitializeTcpipUnapi()
         Terminate("No TCP/IP UNAPI implementations found");
     }
     UnapiBuildCodeBlock(NULL, 1, codeBlock);
-    
+
     regs.Bytes.B = 1;
     UnapiCall(codeBlock, TCPIP_GET_CAPAB, &regs, REGS_MAIN, REGS_MAIN);
     if((regs.Bytes.L & (1 << 3)) == 0) {
         Terminate("This TCP/IP implementation does not support active TCP connections.");
     }
-    
+	if(regs.Bytes.D & 16) //fifth bit is set?

```

Comment:
Please convert that 16 to a constant, e.g. `TCPIP_CAPAB_VERIFY_CERTIFICATE`. This way the code reads better and you don't need the comment.

Reply:
Done :)